app/Console/Commands/RosterUpdate.php

```diff
@@ -139,4 +144,44 @@ public function handle() {
             $u->save();
         }
     }
+
+    private function updateAcademyExams($user) {
```

Comment:
I understand what you are doing here and I appreciate that this transfers the burden for the API call from the front-end to the back-end. I ran this task in QA and it has increased the runtime for RosterUpdate from < 1 minute to around 10 minutes!

Reply:
Just a thought... to relieve some unnecessary runtime on this task, do you think we could add something like:
`if (in_array($user->rating_id, ['C1','C3','I1','I3'])) {
      return;
   }
`
Can you think of any reason that we should still check for people that have a C1 rating or above?

Reply:
Yeah, really surprised me too. I wonder if VATUSA is rate-limiting the API calls which is causing them to take so long to execute. I guess maybe they would just fail in that case... I don't know.

Reply:
> I think that would be fine because even if we don't get something back, those people won't be training anyways. Also should we ignore people that are not active on the roster?
>
Yes, let's ignore inactive members.

Reply:
@kjporter Please take another look. I made the following optimizations and it appeared to speed it up quite a bit:

1. C1, C3, I1, I3 controllers are skipped
2. Inactive and LOA controllers are skipped
3. Controller's exams are updated once every 2 hours. This compares `controller_id mod(2)` to `hour mod(2)` in order to detect whether or not to update the controller. This may not be necessary - I just added it to explore if it speeds up substantially or not

Let me know what you think